django/api/views.py

```diff
@@ -27,9 +27,10 @@
 from rest_framework.exceptions import ValidationError
 from django.db import IntegrityError
 
+
 class NestedMultipartParser(parsers.MultiPartParser):
     def parse(self, stream, media_type=None, parser_context=None):
-        result = super().parse(stream=stream, media_type=media_type, parser_context=parser_context)
+        result = super(NestedMultipartParser, self).parse(stream=stream, media_type=media_type, parser_context=parser_context)
```

Comment:
why is this change here?

Reply:
Oh that was just a driveby change for python compatibility, but does not make sense in case of preconfigured docker container, so I drop this